virtcontainers/interfaces.go

```diff
@@ -69,6 +73,10 @@ type VCSandbox interface {
 	SignalProcess(containerID, processID string, signal syscall.Signal, all bool) error
 	WinsizeProcess(containerID, processID string, height, width uint32) error
 	IOStream(containerID, processID string) (io.WriteCloser, io.Reader, io.Reader, error)
+	AddNetwork(inf grpc.Interface) error
+	ListNetwork() ([]grpc.Interface, []*grpc.Route, error)
+	UpdateNetwork(interfaces []grpc.Interface) error
+	UpdateRoute(routes []*grpc.Route) error
```

Comment:
I guess this should align with VC interfaces:
```
+	ListNetwork() ([]grpc.Interface, []*grpc.Route, error)
+	UpdateNetwork(interfaces []grpc.Interface, routes []*grpc.Route) error
```

Reply:
Refer to https://github.com/kata-containers/documentation/blob/master/design/kata-api-design.md#sandbox-hotplug-api. Though it is a little wired?

Reply:
I agree that we should modify the design docs instead